config/site.toml

```diff
@@ -1,5 +1,5 @@
 title = "Spin Docs"
-# base_url = "https://spinframework.com"
+# base_url = "https://spinframework.dev"
```

Comment:
Reposting: https://github.com/spinframework/spin-docs/pull/13#discussion_r2011094135  cc @karthik2804

Reply:
This can be safely removed.